packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -429,53 +399,63 @@ class Treemap extends StatelessWidget {
 
   Widget buildTitleText(BuildContext context) {
     if (isOutermostLevel) {
-      final pathFromRoot = rootNode.pathFromRoot();
-      // Build breadcrumbs navigator.
+      return buildBreadcrumbsNavigator();
+    } else {
       return Container(
         height: treeMapHeaderHeight,
-        child: ListView.separated(
-          shrinkWrap: true,
-          scrollDirection: Axis.horizontal,
-          separatorBuilder: (context, index) {
-            return const Text(' > ');
-          },
-          itemCount: pathFromRoot.length,
-          itemBuilder: (BuildContext context, int index) {
-            return buildSelectable(
-              child: Text(
-                index < pathFromRoot.length - 1
-                    ? pathFromRoot[index].name
-                    : pathFromRoot[index].displayText(),
-              ),
-              newRoot: pathFromRoot[index],
-            );
-          },
+        width: double.infinity,
+        decoration: BoxDecoration(
+          border: Border.all(color: Colors.black87),
         ),
-      );
-    } else {
-      return buildSelectable(
-        child: Container(
-          height: treeMapHeaderHeight,
-          width: double.infinity,
-          decoration: BoxDecoration(
-            border: Border.all(color: Colors.black87),
-          ),
-          child: buildNameAndSizeText(
-            textColor: Theme.of(context).textTheme.bodyText2.color,
-            oneLine: true,
-          ),
+        child: Text(
+          rootNode.displayText(),
+          style: TextStyle(color: Theme.of(context).textTheme.bodyText2.color),
```

Comment:
this should be the same color you are using in the cell painters (Colors.black87). Either put in a const or use the color from theme in the painter

Reply:
So the `Colors.black87` is for drawing the cell border, and the current font color for cell painter is `showDiff ? Colors.white : Colors.black` to ensure the text is visible depending on the diff color. For this specific line, we're building the text for the title bar, which doesn't depend on the diff color but the theme.